<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MissionRecap</class>
 <widget class="QWidget" name="MissionRecap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1620</width>
    <height>780</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>74</green>
        <blue>84</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>74</green>
        <blue>84</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>74</green>
        <blue>84</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>67</red>
        <green>74</green>
        <blue>84</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="redoMission">
     <property name="styleSheet">
      <string notr="true"> QPushButton { 
background-color: &quot;#CCD1D9&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
     </property>
     <property name="text">
      <string>Redo Mission</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="newMission">
     <property name="styleSheet">
      <string notr="true"> QPushButton { 
background-color: &quot;#CCD1D9&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
     </property>
     <property name="text">
      <string>New Mission</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="replayMission">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>67</red>
           <green>74</green>
           <blue>84</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>204</red>
           <green>209</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>67</red>
           <green>74</green>
           <blue>84</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>67</red>
           <green>74</green>
           <blue>84</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>204</red>
           <green>209</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>204</red>
           <green>209</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>67</red>
           <green>74</green>
           <blue>84</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>204</red>
           <green>209</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>67</red>
           <green>74</green>
           <blue>84</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>67</red>
           <green>74</green>
           <blue>84</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>204</red>
           <green>209</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>204</red>
           <green>209</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>67</red>
           <green>74</green>
           <blue>84</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>204</red>
           <green>209</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>67</red>
           <green>74</green>
           <blue>84</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>67</red>
           <green>74</green>
           <blue>84</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>204</red>
           <green>209</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>204</red>
           <green>209</green>
           <blue>217</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton { 
background-color: &quot;#CCD1D9&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
     </property>
     <property name="text">
      <string>Replay Mission</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QFrame" name="frame_5">
     <property name="styleSheet">
      <string notr="true"> QFrame, QLabel, QToolTip {
     border: 2px solid grey;
     border-radius: 4px;
     padding: 2px;
	 color: #434A54;
	 font-weight: bold;
     background-color: #CCD1D9;
 }</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>1490</x>
        <y>340</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QPushButton { 
background-color: #434A54;
color: &quot;#ffffff&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #656D78
 }</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="playButton">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>340</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QPushButton { 
background-color: #434A54;
color: &quot;#ffffff&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #656D78
 }</string>
      </property>
      <property name="text">
       <string>Play/Pause</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopButton">
      <property name="geometry">
       <rect>
        <x>1240</x>
        <y>340</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QPushButton { 
background-color: #434A54;
color: &quot;#ffffff&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #656D78
 }</string>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openFileButton">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>340</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QPushButton { 
background-color: #434A54;
color: &quot;#ffffff&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #656D78
 }</string>
      </property>
      <property name="text">
       <string>Open File</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>781</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QFrame, QLabel, QToolTip {
     border: 2px solid grey;
     border-radius: 4px;
     padding: 2px;
	 color: #434A54;
	 font-weight: bold;
     background-color: #CCD1D9;
 }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCustomPlot" name="customPlot" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>761</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>10</y>
        <width>781</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QFrame, QLabel, QToolTip {
     border: 2px solid grey;
     border-radius: 4px;
     padding: 2px;
	 color: #434A54;
	 font-weight: bold;
     background-color: #CCD1D9;
 }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QVideoWidget" name="VideoWidget" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>761</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>400</y>
        <width>781</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QFrame, QLabel, QToolTip {
     border: 2px solid grey;
     border-radius: 4px;
     padding: 2px;
	 color: #434A54;
	 font-weight: bold;
     background-color: #CCD1D9;
 }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>761</width>
         <height>311</height>
        </rect>
       </property>
       <property name="plainText">
        <string>                                         Status</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>781</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> QFrame, QLabel, QToolTip {
     border: 2px solid grey;
     border-radius: 4px;
     padding: 2px;
	 color: #434A54;
	 font-weight: bold;
     background-color: #CCD1D9;
 }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWebView" name="webView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>761</width>
         <height>311</height>
        </rect>
       </property>
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>320</y>
        <width>771</width>
        <height>21</height>
       </rect>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>5000</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>360</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Line</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pie</string>
       </property>
      </item>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Graph Type:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>360</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;X Parameters:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_3">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>360</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Y Parameters:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>360</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Hours</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Distance</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>360</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Work Completed</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Power</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="backButton">
     <property name="styleSheet">
      <string notr="true"> QPushButton { 
background-color: &quot;#CCD1D9&quot;; 
color: &quot;#434A54&quot;; 
border-radius: 10px;
font-weight: bold;
border: none; 
}
 QPushButton:pressed {
     background-color: #E6E9ED
 }</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>replayMission</sender>
   <signal>clicked()</signal>
   <receiver>MissionRecap</receiver>
   <slot>replayMissionClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>566</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>replayMissionClicked()</slot>
 </slots>
</ui>
